tests/test_pytypes.py

```diff
@@ -244,6 +244,20 @@ def __repr__(self):
         m.str_from_string_from_str(ucs_surrogates_str)
 
 
+@pytest.mark.parametrize(
+    "func",
+    [
+        m.str_from_bytes_input,
+        m.str_from_cstr_input,
+        m.str_from_string_input,
```

Comment:
```suggestion
        m.str_from_std_string_input,
```

If you did want to change it.